torch_xla/csrc/aten_xla_type.h

```diff
@@ -31,6 +31,11 @@ class AtenXlaType {
 
   static at::Tensor __rshift__(const at::Tensor& self, const at::Tensor& other);
 
+  static at::Tensor adaptive_avg_pool3d(const at::Tensor& self,
+                                        at::IntArrayRef output_size);
+
+  static at::Tensor adaptive_avg_pool3d_backward(const at::Tensor& grad_output,
+                                                 const at::Tensor& self);
```

Comment:
One empty line here ... sorry I missed this.

Reply:
No problem sir. added it now.